tests/sentry/api/endpoints/test_event_attachments.py

```diff
@@ -41,3 +41,36 @@ def test_simple(self):
         assert len(response.data) == 1
         assert response.data[0]["id"] == str(attachment1.id)
         assert response.data[0]["mimetype"] == attachment1.mimetype
+
+    def test_is_screenshot(self):
+        self.login_as(user=self.user)
+
+        min_ago = iso_format(before_now(minutes=1))
+        event1 = self.store_event(
+            data={"fingerprint": ["group1"], "timestamp": min_ago}, project_id=self.project.id
+        )
+
+        attachment1 = EventAttachment.objects.create(
```

Comment:
There's a `create_event_attachment` fixture. Should be accessible in the TestCase class.

Reply:
I think it'd be more appropriate to adjust all of the attachment tests at once with the fixture rather than as a part of this PR